library/nametab.ml

```diff
@@ -279,10 +279,10 @@ module ExtRefTab = Make(FullPath)(ExtRefEqual)
 module MPTab = Make(FullPath)(MPEqual)
 
 type ccitab = ExtRefTab.t
-let the_ccitab = ref (ExtRefTab.empty : ccitab)
+let the_ccitab = Summary.ref ~name:"ccitab" (ExtRefTab.empty : ccitab)
```

Comment:
Very minor nit, maybe a structured naming `tab:cci` would be nicer. Understanding the summary is essential in work with workers so every bit helps.